code/controllers/subsystem/movement/movement_types.dm

```diff
@@ -44,6 +44,12 @@
 	src.lifetime = timeout
 	return TRUE
 
+///proc that exists so we can check if this exact moveloop datum already exists (in terms of vars) and so we can stop it from needlessly create a new one to overwrite the old one
```

Comment:
Mild nitpick, but I think something like this might be a bit easier to read.
Main thing is just removing the "proc that exists so we can", since that's restating info we already know.
```suggestion
///check if this exact moveloop datum already exists (in terms of vars) so we can avoid creating a new one to overwrite the old duplicate
```